source/grid/tria.cc

```diff
@@ -12333,28 +12333,18 @@ std::vector<types::boundary_id> Triangulation<dim, spacedim>::get_boundary_ids()
 {
   // in 1d, we store a map of all used boundary indicators. use it for
   // our purposes
+  std::set<types::boundary_id> boundary_ids;
   if (dim == 1)
```

Comment:
The comment applies to the `if (dim==1)`, not the variable right after it. Just move the variable declaration above the comment.

Reply:
That makes more sense than what I wrote but it turns out we can just delete the whole 1d block.